GUI/Makefile

```diff
@@ -164,6 +163,9 @@ $(DistDir)/$(EXEName): $(Objects) $(DLLs) $(Fonts)
 $(BuildDir)/obj/%.o: %.cpp $(Headers)
 	mkdir -p $(@D)
 	$(CXX) $(Options) -c $(Includes) $< -o $@
+
+$(Engine)/obj/libEngine.a: FORCE
+	make -C $(Engine)

```

Comment:
@happycoder97 

You can remove this again if it's causing you issues -- I just find it convenient to automatically rebuilding the library when something changes in the engine.  To get it to work like this on Windows, you need to copy (or symbolic link) `mingw32-make.exe` to `make.exe`.

Reply:
Okay